app/components/post-meta.js

```diff
@@ -1,5 +1,18 @@
 import Ember from 'ember';
 
+/**
+  The post-meta component is a badge that presents a post's status
```

Comment:
That's a fair point. I think most of the components here were pretty clear what they were just by naming whereas this one took a small amount of digging. I'll look into it this evening.